src/App.jsx

```diff
@@ -14,45 +17,72 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_ALF = 'alphabetically';
+const SORT_BY_LEN = 'length';
+
+function getPreparedGoods(sortBy, reversed) {
+  const preparedGoods = [...goodsFromServer];
+
+  if (sortBy === SORT_BY_ALF) {
+    preparedGoods.sort();
+  } else if (sortBy === SORT_BY_LEN) {
+    preparedGoods.sort((a, b) => a.length - b.length);
+  }
+
+  if (reversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
![image](https://github.com/mate-academy/react_list-of-goods-js/assets/96409563/8fe81dd3-cf28-48ea-9ed9-fdac38ba9f7f)